### PR TITLE
added .env.example.docker

### PR DESCRIPTION
**Pull request by FloxD**
the .env.example.docker can be used if you follow the instructions from https://github.com/esmBot/esmBot/wiki/Setup#docker
the only difference is that the ip for postgres is already set to 172.20.0.5
the same ip that is set in the docker-compose.yml

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -29,7 +29,6 @@ typings/
 .node_repl_history
 *.tgz
 .yarn-integrity
-.env
```
Comment (FloxD) [inline .gitignore:32]: removed this because it's duplicated

File: .gitignore [hunk 2]
```diff
@@ -38,4 +37,5 @@ bannedusers.json
 todo.txt
 .vscode/
 migratedb.js
-processed.txt
\ No newline at end of file
+processed.txt
+.idea/
```
Comment (FloxD) [inline .gitignore:41]: project folder that is created by default for intellij users

File: .env.example.docker [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+NODE_ENV=development
+
+# Put Discord token here
+TOKEN=
+
+# Put the database connection URL here
+# Example for MongoDB:
+# DB=mongodb://localhost:27017/esmBot
+# Example for PostgreSQL:
+DB=postgresql://esmbot:verycoolpass100@172.20.0.5:5432/esmbot
```
Comment (FloxD) [inline .env.example.docker:15]: this is the only difference to .env.example

**Comment by TheEssem** [comment]
Having two effectively duplicate files that serve the same purpose does nothing but clog up the codebase, especially when the change is a simple one.

**Comment by FloxD** [comment]
i got 2 other suggestions in that case:

1.
changing
`DB=postgresql://esmbot:verycoolpass100@postgres:5432/esmbot`
to
`DB=postgresql://esmbot:verycoolpass100@172.20.0.5:5432/esmbot`
in https://github.com/esmBot/esmBot/wiki/Setup#docker

2.
making 172.20.0.5 the default in the normal .env.example

**Comment by TheEssem** [comment]
`postgres` essentially acts as an alias for the actual IP when using docker-compose, so using the actual IP is unnecessary in most cases.

**Comment by FloxD** [comment]
wait really?
didnt know that was a thing

**Pull request closed without merging**